src/witan/models/dem/ccm/core/projection_loop.clj

```diff
@@ -31,12 +31,12 @@
    :witan/version "1.0"
    :witan/input-schema {:population PopulationSchema}
    :witan/param-schema {:_ nil}
-   :witan/output-schema {:latest-yr-popn PopulationSchema}}
+   :witan/output-schema {:latest-yr-popn PopulationSchema :loop-year s/Int}}
   [{:keys [population]} _]
   (let [latest-yr (get-last-yr-from-popn population)
```

Comment:
This can be calculated once before the loop kicks off, and then just incremented each time round.